app/views/docs/authentication.phtml

```diff
@@ -0,0 +1,278 @@
+</p>
+
+<div class="ide" data-lang="javascript" data-lang-label="Web SDK">
+    <pre class="line-numbers"><code class="prism language-javascript" data-prism>import { Client, Account } from "appwrite";
+
+const client = new Client();
+
+const account = new Account(client);
+
+client
+    .setEndpoint('https://[HOSTNAME_OR_IP]/v1') // Your API Endpoint
+    .setProject('5df5acd0d48c2') // Your project ID
+;
+
+// Go to OAuth provider login page
+account.createOAuth2Session('amazon');
+
+</code></pre>
+</div>
+
+<p>  
+    If there is already an active session, the new session will be attached to the logged-in account. If there are no active sessions, the server will attempt to look for a user with the same email address as the email received from the OAuth2 provider and attach the new session to the existing user. If no matching user is found - the server will create a new user.
+</p>
+
+<h3><a href="/docs/authentication#phone" id="phone">Phone</a></h3>
+
+<p>
+    Phone authentication is done using a two-step authentication process. When using phone authentication, the request to authenticate is initiated from the client application, an SMS is sent to the user with a secret key for creating a session. 
+</p>
+
+<p>
+    The example below shows you how to initiate a phone authentication using the Web SDK.
+</p>
+
+<div class="ide" data-lang="javascript" data-lang-label="Web SDK">
+    <pre class="line-numbers"><code class="prism language-javascript" data-prism>import { Client, Account } from "appwrite";
+
+const client = new Client();
+
+const account = new Account(client);
+
+client
+    .setEndpoint('https://[HOSTNAME_OR_IP]/v1') // Your API Endpoint
+    .setProject('5df5acd0d48c2') // Your project ID
+;
+
+const promise = account.createPhoneSession('[USER_ID]', '');
+
+promise.then(function (response) {
+    console.log(response); // Success
+}, function (error) {
+    console.log(error); // Failure
+});
+
+</code></pre>
+</div>
+
+<p>
+    After initiation, the returned username and secret are used to confirm the user.
+</p>
+
+<div class="ide" data-lang="javascript" data-lang-label="Web SDK">
+    <pre class="line-numbers"><code class="prism language-javascript" data-prism>const promise = account.updatePhoneSession('[USER_ID]', '[SECRET]');
+
+promise.then(function (response) {
+    console.log(response); // Success
+}, function (error) {
+    console.log(error); // Failure
+});
+
+</code></pre>
+</div>
+
+<p>
+    If the provided user ID has not be registered, a new user will be created.
+</p>
+
+<h2><a href="/docs/authentication#login" id="login">Login</a></h2>
+
+<p>
+    Logging in with an email and password is one of the most common ways to login into an application. You can create a session maintained across page reloads and browsing sessions. Appwrite provides email and password authentication out of the box. Using one of Appwrite's client SDKs, or the REST APIs directly, you can manage user login sessions in your application.
```

Comment:
I feel like a lot of this writing is similar to line 25. Can this be written to be shorter and more concise?